vault/token_store.go

```diff
@@ -4,14 +4,13 @@ import (
 	"encoding/json"
 	"fmt"
 	"regexp"
```

Comment:
This file should not have been changed for this PR. Had an issue switching between branches.